CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: For Multiworld, sending collected locations to the server can no longer fail if there's an error encoding the inventory.
 - Fixed: Map tracker selects the correct start location if the preset has only one start location that is not the default. 
 
+### Auto-Tracker Pixel Icons
+- Metroid Prime: Visors are now pilled, Boost Ball icon with a proper trail, improvements to Power Bomb icon
+- Metroid Prime 2: Echoes: Visors are now pilled, Boost Ball icon with a proper trail, Screw Attack icon now faces clockwise, dedicated Power Beam icon
```

Comment:
Please end messages with a dot.

Also fwiw, I would've been totally fine if you just phrased them as "Cleaned up and improved icons for Prime 1 and Prime 2" without going into full detail :p